crates/router/src/connector/adyen.rs

```diff
@@ -688,7 +688,7 @@ impl api::IncomingWebhook for Adyen {
             .change_context(errors::ConnectorError::WebhookSourceVerificationFailed)?;
 
         let message = format!(
-            "{}:{}:{}:{}:{}:{}:{}:{}",
+            "{}:{}:{}:{}:{}:{}:{:?}:{}",
```

Comment:
Use a display implementation for `event_code`

Reply:
Okay